### PR TITLE
SolutionDependencyChecker Updates

### PR DESCRIPTION
**Pull request by atraczyk24**

### CODE REVIEW

File: XRMSolutionDependencyChecker/PluginControl.cs [hunk 1]
```diff
@@ -282,7 +301,8 @@ public string ShowMissingComponents(byte[] ExportedSolution)
                     tComp.Columns.Remove("Required Component Parent Schema Name");
                     gridWidth -= 400;
```
Comment (bonomo1) [inline XRMSolutionDependencyChecker/PluginControl.cs:302]: I am a bit confused why it appears that we are hardcoding width?

**Comment by bonomo1** [comment]
@atraczyk24 Can you place the DLL in the nuget folder? 

image

File: XRMSolutionDependencyChecker/PluginControl.cs (hunk 1 above)
Comment (atraczyk24) [reply XRMSolutionDependencyChecker/PluginControl.cs:302]: Forgot to remove the hardcoding of subtracting the width depending on checkboxes (this was there in the original design)

**Comment by bonomo1** [comment]
Update the assembly info. This shows in the tool. Change version to 1.2. Change Heartland Consultants to Corterra Solutions. Change "ProducT" name to Solution Dependency Checker (remove XRM). 
image

**Pull request merged**